roles/openshift_logging/tasks/generate_configmaps.yaml

```diff
@@ -6,8 +6,17 @@
       when: es_logging_contents is undefined
       changed_when: no
 
+    - local_action: >
```

Comment:
I copied this locally because when running the template step, it failed.  I believe its because the module assumes 'src' is a location on the control node and the template module does not support src from memory.